src/known-organizations/index.js

```diff
@@ -2,33 +2,111 @@ import aragonGovernanceImage from './images/aragon-governance.svg'
+        image: aragonGovernanceImage,
+        recommended: true,
+        template: TEMPLATE_DEMOCRACY,
+      },
+      {
+        address: '0x635193983512c621e6a3e15ee1dbf36f0c0db8e0',
+        domain: 'a1.aragonid.eth',
+        name: 'Aragon One',
+        image: aragonOneImage,
+        template: TEMPLATE_DEMOCRACY,
+      },
+      {
+        address: '0x67757A18eDA83125270Ef94dCec7658Eb39bD8a5',
+        domain: 'blankdao.aragonid.eth',
+        name: 'BlankDAO',
+        image: blankDaoImage,
+        recommended: true,
+        template: TEMPLATE_DEMOCRACY,
+      },
+      {
+        address: '0xcD3d9b832BfF15E0a519610372c6AAC651872DdE',
+        domain: '',
+        name: 'MyBit',
+        image: myBitImage,
+        recommended: true,
+        template: TEMPLATE_DEMOCRACY,
+      },
+      {
+        address: '0x0ee165029b09d91a54687041Adbc705F6376C67F',
+        domain: '',
+        name: 'Livepeer',
+        image: livepeerImage,
+        recommended: true,
+        template: TEMPLATE_DEMOCRACY,
+      },
+      {
+        address: '0x5aad137d8f7d2dc6e1b2548c059b1483360bcc6a',
+        domain: 'brightid.aragonid.eth',
+        name: 'BrightID',
+        image: brightIdImage,
+        recommended: true,
+        template: TEMPLATE_DEMOCRACY,
+      },
+    ].map(org => [org.address, org])
+  ),
+  rinkeby: new Map(
     [
-      '0x2de83b50af29678774d5abc4a7cb2a588762f28c',
-      { name: 'Aragon Governance', image: aragonGovernanceImage },
-    ],
+      {
+        address: '0x383037aD4E2341835Bfe719E4Bddc5936c271409',
+        domain: 'governance.aragonproject.eth',
+        name: 'Aragon Governance',
+        image: aragonGovernanceImage,
+        recommended: true,
+        template: TEMPLATE_DEMOCRACY,
+      },
+      {
+        address: '0x837354C9dBEB4169157ef335762367e350e90eEd',
+        domain: 'a1.aragonid.eth',
+        name: 'Aragon One',
+        image: aragonOneImage,
+        recommended: true,
+        template: TEMPLATE_DEMOCRACY,
+      },
+    ].map(org => [org.address, org])
+  ),
+}
 
-    // a1.aragonid.eth
-    [
-      '0x635193983512c621e6a3e15ee1dbf36f0c0db8e0',
-      { name: 'Aragon One', image: aragonOneImage },
-    ],
+// Get the organizations that might appear in the suggestions,
+// using the format `{ address, name }` where name is the ENS domain.
+export const getRecommendedOrganizations = (networkType, max = -1) => {
+  if (!KnownOrganizations[networkType]) {
+    return []
+  }
 
-    // brightid.aragonid.eth
-    [
-      '0x5aad137d8f7d2dc6e1b2548c059b1483360bcc6a',
-      { name: 'BrightID', image: brightIdImage },
-    ],
-  ]),
+  const recommended = []
+  for (const [address, org] of KnownOrganizations[networkType]) {
+    if (org.recommended) {
+      recommended.push({ address, name: org.domain })
+      if (recommended.length === max) {
+        break
+      }
+    }
+  }
+
+  return recommended
 }
 
 export const getKnownOrganization = (networkType, address) => {
```

Comment:
A couple of things for @luisivan and @lkngtn:

- We should reach out to see if Mybit and Livepeer would like to assign a name; it's fairly easy to do so now
- Let's create some more example orgs using the 0.8 templates so we can suggest them to users on mainnet and rinkeby
- In general, the organization type needs a bit of sanity checking; none of these are base "democracy" orgs (they're all custom deployments), and we probably want to move away from the old "democracy" and "multisig" template naming.